pkg/apiserver/apiserver.go

```diff
@@ -40,7 +40,9 @@ var (
 	Scheme = runtime.NewScheme()
 	// Codecs provides methods for retrieving codecs and serializers for specific
 	// versions and content types.
-	Codecs = serializer.NewCodecFactory(Scheme)
+	Codecs          = serializer.NewCodecFactory(Scheme)
+	TokenPath       = "/tmp/controller-loopback-token"
+	TokenByteLength = 64
```

Comment:
No need to have this if you use apiserver generated token.

Reply:
Sure, removed.